hbase22/README.md

```diff
@@ -15,25 +15,17 @@ permissions and limitations under the License. See accompanying
 LICENSE file.
 -->
 
-# HBase (0.98.x) Driver for YCSB
-This driver is a binding for the YCSB facilities to operate against a HBase 0.98.x Server cluster.
-To run against an HBase >= 1.0 cluster, use the `hbase10` binding.
+# HBase (2.1+) Driver for YCSB
```

Comment:
If this works with HBase 2.1.x, why is the module called hbase22? Prior convention?

Reply:
I only wanted to include the versions with stable pointers pointing at them. When I started the PR back in October though there was no stable branch-2, so it was just the latest which was 2.1.

I can update this text to say 2.2+, maybe? In theory `hbase14` is really the "HBase 1.y API" and `hbase22` is really "HBase 2.y API". AFAIK the only minor release specific thing either module makes use of is the shaded testing util artifacts.